app/components/views/PrivacyPage/Privacy/Privacy.module.css

```diff
@@ -35,4 +35,20 @@
 }
 .contentTitleButtonsArea {
   margin-left: auto;
-}
\ No newline at end of file
+}
+.logs{
```

Comment:
Probably need some max height here, and vertical scroll when overflowing

Reply:
I added the logs into a textarea, so it has a similar behavior from logs. Thanks for the catch.